js/core/utils/date.js

```diff
@@ -502,6 +502,14 @@ function getFirstWeekDate(date, firstDayOfWeek) {
     return result;
 }
 
+function getWeekNumber(date, firstDayOfWeekIndex = 0) {
+    const yearFirstDay = new Date(date.getFullYear(), 0, 1);
```

Comment:
Will be it correct with various time zones?

Reply:
As far as `new Date` takes current timezone, I don't think there should be any problems with timezone